docs/doc/60-contributing/03-rfcs/20220809-share.md

```diff
@@ -0,0 +1,171 @@
+
+- Create Share
+- Drop Share
+- Alter Share
+- Grant share object privilege TO Share
+- Revoke shared object privilege FROM Share
+- Show Grants
+- Create Database, Table FROM Share
+
+### Create Share
+
+Create an empty new share. Once the share is created, the user can use the `GRANT <privilege> ... TO SHARE` command to include a database and objects in the database (tables) in the share. Users can then use `ALTER SHARE` to add one or more tenants to that share.
+
+Syntax:
+
+```sql
+CREATE SHARE [IF NOT EXISTS] <share_name> [ COMMENT = '<string_literal>' ]
+```
+
+### Drop Share
+
+Remove the specified share from the system and immediately revoke access to all tenants (i.e., the tenants that created the database from that share).
+
+Syntax:
+
+```sql
+DROP SHARE [IF EXISTS] <share_name>
+```
+
+### Alter Share
+
+Modify the properties of an existing share.
+
+- Add or remove tenants from the tenant list.
+
+Syntax:
+
+```sql
+ALTER SHARE [IF EXISTS] <share_name> {ADD | REMOVE} TENANTS = <tenant_name> [, <tenant_name>, ...]
+```
+
+### Grant share object privilege TO SHARE
+
+Grant access to the database and other supported database objects (tables, views, etc.) to a share. Granting permissions to these objects effectively adds them to the share, which can then be shared with one or more tenants.
+
+Syntax:
+
+```sql
+GRANT object_privilege ON grant_share_object_name TO SHARE <share_name>
+
+object_privilege := [USAGE|SELECT|REFERENCE_USAGE]
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+### Revoke share object privilege FROM SHARE
+
+Revoke access to databases and other supported database objects (tables, views, etc.) in the share. Revoking permissions to these objects effectively removes them from the share, making them inaccessible to all tenants that have created databases in the share.
+
+Syntax:
+
+```sql
+REVOKE object_privilege ON grant_share_object_name FROM SHARE <share_name>
+
+object_privilege := [USAGE|SELECT|REFERENCE_USAGE]
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+### Show Grants
+
+Lists all access control privileges that have been explicitly granted to tenants and shares.
+
+Syntax:
+
+```sql
+SHOW GRANTS
+SHOW GRANTS ON grant_share_object_name
+SHOW GRANTS TO {TENANT <tenant_name> | SHARE <share_name> }
+SHOW GRANTS OF SHARE <share_name>
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+- `SHOW GRANTS`: Syntactically equivalent to SHOW GRANTS TO TENANT current_tenant. Lists all the roles granted to the current tenant.
+- `SHOW GRANTS ON grant_share_object_name`: Lists all privileges that have been granted on the object.
+- `SHOW GRANTS TO ...`:
+  - `TENANT <tenant_name>`: Lists all the privileges granted to the tenant.
+  - `SHARE <share_name>`: Lists all the privileges granted to the share.
```

Comment:
Is the term `privilege` the same as `object_privilege` at line-87?

Reply:
yes.